showkase-processor-testing/src/androidTest/java/com/airbnb/android/showkase_processor_testing/ShowkaseBrowserTestFlows.kt

```diff
@@ -0,0 +1,77 @@
+package com.airbnb.android.showkase_processor_testing
+
+import androidx.ui.test.android.AndroidComposeTestRule
+import androidx.ui.test.assertCountEquals
+import androidx.ui.test.assertIsDisplayed
+import androidx.ui.test.onChildren
+import androidx.ui.test.onNodeWithSubstring
+import androidx.ui.test.onNodeWithTag
+import androidx.ui.test.onNodeWithText
+import androidx.ui.test.performClick
+import androidx.ui.test.performGesture
+import androidx.ui.test.performTextInput
+import androidx.ui.test.swipeUp
+import androidx.ui.test.waitForIdle
+import com.airbnb.android.showkase.ui.ShowkaseBrowserActivity
+
+internal fun clickRowWithText(text: String) {
+    onNodeWithText(text).assertIsDisplayed().performClick()
+}
+
+internal fun clickRowWithTag(text: String) {
+    onNodeWithTag(text).assertIsDisplayed().performClick()
+}
+
+internal fun inputTextWithTag(tag: String, text: String) {
+    onNodeWithTag(tag).performTextInput(text)
+}
+
+internal fun verifyRowsWithTextAreDisplayed(textList: List<String>) {
```

Comment:
you could potentially make the invocation simpler by making these vararg instead